src/Data/Functor/Alt.hs

```diff
@@ -321,3 +337,21 @@ instance Alt Monoid.First where
 
 instance Alt Monoid.Last where
   (<!>) = mappend
+
+-- Helpers
+
+-- Required to work around https://hub.darcs.net/ross/transformers/issue/67
```

Comment:
In particular, also note that using the official `*.CPS` API would result in unnecessary `Monoid` constraints, which is the motivation for this workaround.